### PR TITLE
Release 1.0.5

### PR DESCRIPTION
**Pull request by malarzm**

### CODE REVIEW

**Comment by alcaeus** [comment]
:+1:

File: CHANGELOG-1.0.md [hunk 1]
```diff
@@ -12,8 +12,20 @@ the older and newer versions, respectively.
 To generate a changelog summary since the last version, run
 `git log --no-merges --oneline XXX...HEAD`
 
-1.0.x-dev
----------
+1.0.5 (2016-02-16)
```
Comment (jmikola) [inline CHANGELOG-1.0.md]: Should we leave the `1.0.x-dev` line at the top? I think the originally intention was to fill this in as PRs were written, but if that's not needed we can do without it.

File: docs/en/conf.py [hunk 2]
```diff
@@ -47,7 +47,7 @@
 # The short X.Y version.
 version = '1.0'
 # The full version, including alpha/beta/rc tags.
-release = '1.0.3'
+release = '1.0.5'
```
Comment (jmikola) [inline docs/en/conf.py]: LOL at forgetting the 1.0.4 bump. Classic ODM :)

**Comment by jmikola** [comment]
:+1:

File: CHANGELOG-1.0.md (hunk 1 above)
Comment (malarzm) [inline CHANGELOG-1.0.md]: I saw no changes at least since I'm here, we were always writing up changelog before release

Comment (alcaeus) [inline CHANGELOG-1.0.md]: Same here. It might be useful, it's just that nobody has done it ;)

**Pull request merged**